recipes/bioconductor-simpleaffy/build.sh

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
thanks @bgruening 

FYI: these empty lines are added by `scripts/bioconductor_skeleton.py` so should we fix that there as well?